dace/transformation/auto/auto_optimize.py

```diff
@@ -334,7 +334,7 @@ def find_fast_library(device: dtypes.DeviceType) -> List[str]:
         if openblas.OpenBLAS.is_installed():
             result.append('OpenBLAS')
 
-        return result + ['OpenMP', 'pure']
+        return result + ['pure']
```

Comment:
My understanding is that we have an issue with the OpenMP ReduceNode expansion. However, this change will basically disable in the auto-optimizer all OpenMP expansions. We probably don't have many OpenMP expansions now, but perhaps we should amend this in the future.

Reply:
true. right now the only OpenMP implementation is this reduce node, and openmp is not well supported across compilers :( we had several complaints about this failing.